src/lib.rs

```diff
@@ -478,9 +481,7 @@ fn link_is_publishable(link: &PackageLink) -> bool {
 fn package_is_publishable(pkg: &PackageMetadata) -> bool {
     let result = match pkg.publish() {
         guppy::graph::PackagePublish::Unrestricted => true,
-        guppy::graph::PackagePublish::Registries(registries) => {
-            registries.len() == 1 && registries[0] == "crates.io"
-        }
+        guppy::graph::PackagePublish::Registries(registries) => _registries.len() == 1,
```

Comment:
This is a change worth calling out. This removes the hard dependency on crates.io being the only valid upstream. I imagine we may want to add some other conditions to this. i.e. Maybe checking for auth tokens. for a particular index?